doc/machine-config/windows-dev-env.md

```diff
@@ -2,6 +2,10 @@
 
 Configuration steps, scripts and tools i use on windows machines. Feel free to skip any steps that don't work for you or add steps if you think something is missing.
 
+### Install Visual Studio, Azure SDK, etc
+
+Please install Visual Studio, Azure SDK, etc by this [here](https://github.com/Azure/azure-iot-sdks/blob/master/csharp/device/doc/devbox_setup.md).
```

Comment:
Nit: (I generally prefix my comments with Nit: to indicate that I am being Nitpicky and I'm just correcting grammar or style. You should feel free to ignore any of my comments prefixed with Nit:)

, etc by this [here]

might sound better this way

, etc using the steps [here]